tests/deployment/multi_networks.py

```diff
@@ -0,0 +1,147 @@
+"""Test various properties re: vlan tagged networks, on real hardware.
+
+For guidance on running these tests, see the section on deployment
+tests in docs/testing.md
+"""
+
+import json
+
+from hil import api, model, deferred, config
+from hil.test_common import config_testsuite, fail_on_log_warnings, \
+    fresh_database, with_request_context, site_layout, NetworkTest, \
+    network_create_simple, server_init
+import pytest
+
+
+@pytest.fixture
+def configure():
+    """Configure HIL."""
+    config_testsuite()
+    config.load_extensions()
+
+
+fail_on_log_warnings = pytest.fixture(autouse=True)(fail_on_log_warnings)
+fresh_database = pytest.fixture(fresh_database)
+server_init = pytest.fixture(server_init)
+
+
+with_request_context = pytest.yield_fixture(with_request_context)
+
+site_layout = pytest.fixture(site_layout)
+
+pytestmark = pytest.mark.usefixtures('configure',
+                                     'server_init',
+                                     'fresh_database',
+                                     'with_request_context',
+                                     'site_layout')
+
+
+class TestMultiNets(NetworkTest):
+    """NetworkTest using multiple tagged vlan networks on a port."""
+
+    def test_multi_networks(self):
+        """A suite of tests to ensure that multiple networks can relate
+        to a single port. Previously, there was an issue with
```

Comment:
Would be good to add a reference to the specific Github issue.

Reply:
Should be all set now